insns.def

```diff
@@ -841,23 +846,18 @@ invokeblock
 (...)
 (VALUE val)
 // attr bool handles_sp = true;
-// attr rb_snum_t sp_inc = 1 - ci->orig_argc;
+// attr rb_snum_t sp_inc = sp_inc_of_invokeblock(ci);
 {
-    struct rb_calling_info calling;
-    VALUE block_handler;
+    static struct rb_call_cache cc = {
```

Comment:
trivial: If I understand correctly, effectively it's not used as "cache" here. We may want to rename "call cache" to something indicating that it may not be used as cache, if we do this.